modules/accounts/email.nix

```diff
@@ -133,6 +133,21 @@ let
           Configuration for secure connections.
         '';
       };
+
+      authentication = mkOption {
+        type = types.enum [
+          "password"
+          "encrypted-password"
+          "gssapi"
+          "ntlm"
+          "tls"
+          "oauth2"
```

Comment:
Don't these other method require some additional information? Like a client certificate for the tls one, for example?